### DISCUSSION TITLE
How to build Hybrid Search using Qdrant?

### DISCUSSION THREAD

**Discussion by karndeepsingh** [qdrant/qdrant#1484] [General]
Hi,
I am trying to build hybrid search and I just caught up into Qdrant. I would like to understand following things:
1. Is qdrant free to use?
2. How I can use qdrant for building Hybrid Search? Any documentation or example would be better to get an idea on usage.
3. If I want to deploy this application on GCP using qDrant for Indexing , is it possible to do ??

Thanks

**Comment by generall** [comment]
> Is qdrant free to use?

 yes, it's Apache 2.0 license
 
 > How I can use qdrant for building Hybrid Search? Any documentation or example would be better to get an idea on usage.

 Check out our latest article https://qdrant.tech/articles/hybrid-search/
 
 > If I want to deploy this application on GCP using qDrant for Indexing , is it possible to do ??
 
 Yes, you can serve qdrant along your app in GCP, or access qdrant cloud from GCP

**Reply by karndeepsingh** [reply]
Thanks fore reply

> > Is qdrant free to use?
> 
> yes, it's Apache 2.0 license
> 
> > How I can use qdrant for building Hybrid Search? Any documentation or example would be better to get an idea on usage.
> 
> Check out our latest article https://qdrant.tech/articles/hybrid-search/
> 
> > If I want to deploy this application on GCP using qDrant for Indexing , is it possible to do ??
> 
> Yes, you can serve qdrant along your app in GCP, or access qdrant cloud from GCP

Thanks for replying. 
is there any examples how I can implement Hybrid search with my dense model and sparse model using Qdrant?
It would be a great help if you can share examples of it or any reference that can guide on how to build a hybrid search using qdrant? I guess in article implementation examples are not mentioned.

**Reply by generall** [reply]
We don't have a sparse vectors serving yet, but it is a priority in our roadmap - https://github.com/qdrant/qdrant/blob/master/docs/roadmap/README.md

**Reply by alabrashJr** [reply]
@karndeepsingh I have not found any Hybrid Search related implementation yet, have you?

**Reply by nigh8w0lf** [reply]
Qdrant does not have Hybrid search (Keyword + Vector)  feature for some reason, which is very disappointing ,you will have to Implement it yourself.
This forces me to consider Milvus as an option.
